client/components/Feedback.jsx

```diff
@@ -1,18 +1,54 @@
 import React from 'react';
 import styled from 'styled-components';
 
-import PercentageBar from './PercentageBar';
-import StarRating from './StarRating';
 import RatingSelector from './RatingSelector';
-import Avatar from './Avatar';
 import Review from './Review';
+import SearchBar from './SearchBar';
+import CourseRating from './CourseRating';
 
-const Container = styled.div`
+const StyleProvier = styled.div`
```

Comment:
This might be a typo. I think you meant to put StyleProvider ?

Reply:
fixed!